charts/minecraft/templates/extraports-ing.yaml

```
@@ -0,0 +1,45 @@
+{{- $minecraftFullname := include "minecraft.fullname" . }}
+{{- range .Values.minecraftServer.extraPorts }}
+{{- if default "" .ingress.enabled }}
+{{- $servicePort := .service.port }}
+{{- $serviceName := printf "%s-%s" $minecraftFullname .name }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $serviceName }}
+  {{- if .ingress.annotations }}
+  annotations:
+    {{- toYaml .ingress.annotations | nindent 4 }}
```

I missed this earlier
```suggestion
{{- toYaml .ingress.annotations | nindent 4 }}
```